frontend/src/components/TheStageSlider.vue

```diff
@@ -262,6 +263,7 @@ const stopDrag = () => {
   pointer-events: none;
   box-sizing: border-box;
   width: 100%;
+  height: 400px !important; /* Explicitly set the height */
```

Comment:
for sm screens, the slider will overlap with the plots

<img width="1322" height="2334" alt="Image" src="https://github.com/user-attachments/assets/b282f2da-5108-44d6-8f6f-dd4a868d6ce6" />